.pre-commit-config.yaml

```diff
@@ -7,8 +7,8 @@ repos:
   - repo: https://github.com/ambv/black
     rev: stable
     hooks:
-    - id: black
-      # language_version: python3.6
+      - id: black
```

Comment:
was this indent accidental? if so, is it possible to remove this file from the commit?

Reply:
probably an IDE did that automatically or a linter wanted it.  yaml doesn't care and conventions vary whether list items should be indented two spaces or not.  i note all the other blocks in this file _do_ indent list items so this change makes it consistent.  i'd recommend keeping but happy to remove if you prefer.